src/main/java/org/spongepowered/api/map/MapView.java

```diff
@@ -0,0 +1,242 @@
+
+    /**
+     * Sets the {@link MapCursor} used for item frame locations on the map when
+     * {@link #isUsingVanillaCursors()} is true. By default this is a
+     * {@link MapCursors#GREEN_POINTER}.
+     *
+     * @param cursor The new cursor for item frame positions
+     */
+    void setItemFrameCursor(MapCursor cursor);
+
+    /**
+     * Sets the {@link MapCursor} used for player locations on the edge
+     * when players go out of bounds when  {@link #isUsingVanillaCursors()} is
+     * true. By default this is a {@link MapCursors#WHITE_CIRCLE}.
+     *
+     * @param cursor The new cursor for edge positions
+     */
+    void setEdgeCursor(MapCursor cursor);
+
+    /**
+     * Adds a new cursor to the map of the specified type, the position is pixel
+     * relative and refers to the center position of the cursor.
+     *
+     * @param type The type of cursor to add
+     * @param position The map pixel relative center position of the cursor
+     * @return A handle to the cursor that can be used for updating, removal and
+     *         changing the type of cursor
+     */
+    MapCursorHandle addCursor(MapCursor type, Vector2i position);
+
+    /**
+     * Gets the current scaling ratio of the map.
+     *
+     * @return The scaling ratio for the map
+     */
+    MapScale getScale();
+
+    /**
+     * Sets the current scaling ratio of the map.
+     *
+     * @param scale The new scaling ratio for the map
+     */
+    void setScale(MapScale scale);
+
+    /**
+     * Sends a forced update of the map to a player. This will send only the
+     * region of the canvas between min x,y and max x,y to the player. Allowing
+     * custom trickle loading, or forced immediate updating of the map.
+     *
+     * @param player The player to send the update to
+     * @param min The upper left corner of the region to send
+     * @param max The bottom right corner of the region to send
+     * @throws IndexOutOfBoundsException If the update region is not within the boundaries of the map
+     */
+    void sendUpdate(Player player, Vector2i min, Vector2i max);
+
+    /**
+     * Sends a forced update of the map to multiple players. This will send only the
+     * region of the canvas between min x,y and max x,y to the players. Allowing
+     * custom trickle loading, or forced immediate updating of the map.
+     *
+     * @param players The players to send the updates to
+     * @param min The upper left corner of the region to send
+     * @param max The bottom right corner of the region to send
+     * @throws IndexOutOfBoundsException If the update region is not within the boundaries of the map
+     */
+    void sendUpdate(Collection<Player> players, Vector2i min, Vector2i max);
+
+
+    /**
+     * Returns if the vanilla behavior for map updates (slowly updating a few
+     * rows) at a time is being used.
+     *
+     * @return True if vanilla behavior is being used, false otherwise
+     */
+    boolean isDoingAutomaticUpdates();
```

Comment:
`autoUpdates()`? Not every `boolean` method needs to be prefixed with `is`. IMO as long as a condition can be read like english it's fine and this gets the same point across with less verbosity.

```java
if (mapView.autoUpdates()) // "If the map view auto updates."
```